### DISCUSSION TITLE
How docusaurus supports micro front ends(eg:qiankun, single-spa)

### DISCUSSION THREAD

**Discussion by kezhongfa** [facebook/docusaurus#9066] [General]
Portal: want to collect together several single applications generated by docusaurus

**Comment by baretta881** [comment]
{
  "prefix": "-moz-",
  "version_added": "3.5"
}

**Comment by slorber** [comment]
We don't particularly support microfrontends, and I'm not sure to understand what it even means in the context of Docusaurus 😅 

Show me a mockup of your desired UI
